pwiz_tools/Skyline/Model/Results/SpectrumFilterPair.cs

```diff
@@ -584,10 +597,62 @@ public override int GetHashCode()
                 var hashCode = TargetMz.GetHashCode();
                 hashCode = (hashCode*397) ^ FilterWidth.GetHashCode();
                 hashCode = (hashCode*397) ^ FilterId;
+                hashCode = (hashCode * 397) ^ HighEnergyIonMobilityValueOffset.GetHashCode();
                 return hashCode;
             }
         }
 
         #endregion
     }
+
+    public class SpectrumFilterValues : IComparable<SpectrumFilterValues>, IComparable

```

Comment:
This class only needs to exist if we are going to allow a SpectrumFilterPair to extract the same product m/z with two different high energy offsets.

However, if the m/z no longer uniquely identifies the chromatogram, then we probably need to add the high energy offset value to the class "ChromTransition" so that we will be able to use the correct chromatogram later on.